node/src/chain_spec.rs

```diff
@@ -72,6 +73,20 @@ pub fn development_config() -> Result<ChainSpec, String> {
 				"0xec00ad0ec6eeb271a9689888f644d9262016a26a25314ff4ff5d756404c44112"
 					.parse()
 					.unwrap(),
+				// SnowBridged Assets
+				vec![
+						(
+							"Mangata".as_bytes().to_vec(), "MNG".as_bytes().to_vec(), "Mangata Asset".as_bytes().to_vec(),
+							18u32, 0u32, H160::from_slice(&hex!["F8F7758FbcEfd546eAEff7dE24AFf666B6228e73"][..]),
```

Comment:
Is it mangata address for the local testnet or for Kovan?

Reply:
All addresses are for local testnet.

Reply:
Can you change the address for development chainspec to Kovan mangata address (when it will be deployed), and leave the local address for the local testnet?

Reply:
I have configured everything to run with --dev as local doesn't seem to work.
When you start the node with the local config, it just keeps printing out:
INFO 💤 Idle (0 peers), best: #0 (0x5817…d9ad), finalized #0 (0x5817…d9ad), ⬇ 0 ⬆ 0

Reply:
I see. Please revert the parameters of the dev to match the public testnet since it is the configuration currently used there.